### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by LChan14**
For assignment 3 of SF-1

### CODE REVIEW

**Review by alexder204 — COMMENTED** [review]
This code is all good, there wasn't any syntax errors I saw and it's clear what it's about.

File: count-vowels/starter.py [hunk 1]
```diff
@@ -0,0 +1 @@
+def count-vowels(text):
```
Comment (emaj9) [inline count-vowels/starter.py:1]: Don't put dashed in function names in python (notice how the syntax highlighting looks weird on "vowels"

File: count-vowels/meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Count-vowels",
+  "name": "count-vowels",
+  "difficulty": "hard",
+  "author": "Liam Chan",
+  "category": "count"
```
Comment (emaj9) [inline count-vowels/meta.json:6]: incorrect

File: count-vowels/io.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "input": "Hello World",
```
Comment (emaj9) [inline count-vowels/io.json:3]: syntax error on your inputs

File: count-vowels/description.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Given a string of length `n`, return the `number of vowels`. The vowels `a,e,I,o, and u`, lowercase `ONLY`.
```
Comment (emaj9) [inline count-vowels/description.md:1]: this is a really low quality question. definitely not a hard one either.

**Comment by emaj9** [comment]
There are several errors with this submission. Every file contains an error. In it's current state this addition is unacceptable.  You did not pass this assignment. 

> This code is all good, there wasn't any syntax errors I saw and it's clear what it's about.

Really? @alexder204

**Pull request closed without merging**